src/Form/atoms/DropArea.tsx

```diff
@@ -27,6 +27,12 @@ interface IDropArea {
 
 const DropArea : React.FC<IDropArea> = ({height, text, dropCallback, ...props}) => {
   const [inDropZone, setInDropZone] = useState(false);
+  window.addEventListener("dragover",function(e){
+    e.preventDefault();
+  },false);
+  window.addEventListener("drop",function(e){
+    e.preventDefault();
+  },false);
```

Comment:
Please add and remove event listener inside `useEffect`.

Reply:
Done